pkg/k8sclient/k8sclient.go

```diff
@@ -67,10 +63,22 @@ func GetNode(nodeName string, kubeClient kubernetes.Interface) (*v1.Node, error)
 }
 
 func UpdateNodeStatus(nodeName string, kubeClient kubernetes.Interface, node *v1.Node) error {
-	updatenode, err := kubeClient.CoreV1().Nodes().UpdateStatus(node)
+	_, err := kubeClient.CoreV1().Nodes().UpdateStatus(node)
 	if err != nil {
 		return err
 	}
-	log.Printf("DEBUG: UpdateNodeStatus: %+v\n", updatenode)
 	return nil
 }
+
+func GetLocalStorageClassName(kubeClient kubernetes.Interface) (string, error){
+	scList, err := kubeClient.StorageV1().StorageClasses().List(metav1.ListOptions{})
+	if err != nil {
+		return "", err
+	}
+	for _, sc := range scList.Items {
+		if sc.Provisioner == LocalScProvisioner {
+			return sc.ObjectMeta.Name, nil
+		}
+	}
+	return "", errors.New("There is no local StorageClass configured!")
```

Comment:
not having any SC is not an error IMO, thus should be handled gracefully
no SC == no matching